storages/domain_style.go

```diff
@@ -0,0 +1,41 @@
+package storages
+
+import (
+	"net/http"
+	"fmt"
+	"github.com/allegro/akubra/log"
+)
+
+const (
+	HOST              = "X-3yeLjyjQNx"
+	BUCKET            = "X-lejK0EpVZy"
```

Comment:
Why? Do we really want unreadable, cryptic and suspicious-looking internal headers?
BTW. there is no such convention in go that consts are uppercase, follow mixed case for all names.

Reply:
@ksitak suggested that we should use a truly random value for the header name. 
We can actually do `X-3yeLjyjQNx-Akubra-Internal-Host` as a compromise

Reply:
X-Akubra-Internal-Host-3yeLjyjQNx will be fair deal

Reply:
Yeah, I suggested to include some hash in that header to minimize risk that one day we'll decide to use some X-Akubra...(whatever) header in a real world and we forget that we overwrite it inside.

Reply:
Fixed